views/albums.hbs

```diff
@@ -0,0 +1,16 @@
+<h1>Here are the albums!</h1>
+
+{{#each albums}}
+  <ul>
+    <li>
+      <a href="/">{{this.name}}</a>
```

Comment:
you dont need to put the artist name in a tag!  it makes sense when this a tag goes to albums of this artist,not home page :)